src/Arcus.Messaging.EventHubs.Core/Extensions/AzureClientFactoryBuilderExtensions.cs

```diff
@@ -20,7 +23,7 @@ public static class AzureClientFactoryBuilderExtensions
         /// <remarks>
         ///   <para>
         ///     Make sure that the Arcus secret store is registered in the application before using this extension (<a href="https://security.arcus-azure.net/features/secret-store">more info</a>)
-        ///     as the Azure EventHubs connection string will be retrieved via the <paramref name="connectionStringSecretName"/>.
+        ///     as the Azure EventHubs connection string will be synchronously retrieved via the <paramref name="connectionStringSecretName"/>.
```

Comment:
Are we going to change the implementation of the secret retrieval so that we fall back to async retrieval if no secret could be retrieved synchronously ?  If so, then shouldn't we remove this warning ?

Reply:
This warning is mostly about the consumer that needs to register the Arcus secret store for this to work. There were some 'synchronously' words there still, so these are now removed.
Fallback on async is implemented here, but not sure if we should do that in Arcus Security, as we could have unexpected results of trying to run asynchronous code synchronously, right?

Reply:
Yes that is right.